app/assets/src/components/ui/controls/search_box_list.scss

```diff
@@ -0,0 +1,54 @@
+@import "~styles/themes/colors";
+
+.searchBoxList {
+  background-color: $white;
+  border: 1px solid $light-grey;
+  display: inline-block;
+  font-size: 13px;
+  min-width: 200px;
+  padding: 20px;
+  text-align: left;
+
+  .searchBox {
+    margin-bottom: 10px;
+  }
+
+  .title {
+    font-weight: 700;
+    margin-bottom: 10px;
+  }
+
+  .listBox {
+    height: 200px;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+      width: 6px;
+    }
+
+    &::-webkit-scrollbar-track {
+      background-color: $lightest-grey;
+      border-radius: 3px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+      background-color: $medium-grey;
+    }
+
+    .listElement {
+      cursor: pointer;
+      height: 24px;
+      margin-right: 10px;
+      white-space: nowrap;
+    }
+
+    .listCheckmark {
+      display: inline-block;
+      width: 20px;
+    }
+
+    .listLabel {
```

Comment:
I think it would be good to make sure the styling of the options here is consistent with the Dropdown items. Things like `letter-spacing: 0.4px` and proper top and bottom padding between items. So that all of our dropdowns look consistent.

Reply:
You got a point, but actually i think the problem is larger than that.
For instance, `letter-spacing` is something that should be consistent everywhere, so probably should not be set at the dropdown level.
I can change padding, but I also think this is different component... not a dropdown. I think eventually, if we decided to go without the checkboxes... we might want to reduce the padding.
Anyway is a topic to sync with Jenn, and maybe for a larger change

Reply:
Yeah, talk to Jenn here too.